README.md

```diff
@@ -10,7 +10,7 @@ The Ruby community's gem host.
 ## Support
 
 <a href="https://rubytogether.org/"><img src="https://rubytogether.org/images/rubies.svg" width=200></a>
-<a href="https://rubycentral.org/"><img src="https://gist.githubusercontent.com/sonalkr132/52a23481c0765b36ce1e909ba678c51a/raw/78e47c9ec77b690322040d4b9b8f460f58196182/Ruby-Central-Logo.png" width=160></a><br/>
+<a href="https://rubycentral.org/"><img src="public/images/ruby_central_logo.png" height=110></a><br/>
```

Comment:
I'm not sure that path is right

Reply:
Are you concerned about adding it public folder? Only another place we have images is vendor/assets. I didn't want to introduce a special location for GitHub repo only images.

Reply:
I was wrong, we already have `doc` folder with erd.svg. moved this to `doc`.

Reply:
Oh I didn't realize this was just a markdown file I thought this was web page content and the path was wrong. 🤦 Doc folder seems fine though.